src/model/PopupOption.ts

```diff
@@ -0,0 +1,4 @@
+export interface PopupOption {
+  iconName: string
+  text: string
+}
```

Comment:
This model should be named PopupOptionModel

If you make iconName optional you will be able to reuse this model for this kind of popups:

![Screenshot 2022-02-24 at 09 32 03](https://user-images.githubusercontent.com/40487012/155487720-a2deb80c-1e0d-4c78-a932-9bc582a1b1cc.png)